Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    [SerializeField] private float maxVelocity = 10f;
+    public float speed = 1f;
+    private bool isGrounded;
+    private bool onSlope;
+    public float jumpFactor = 6f;
```

Comment:
als `public` niet nodig is dan is `[SerializeField] private` beter geld voor alle variabelen hier

Reply:
Zou handig kunnen zijn voor later gebruik.